src/main/java/tc/oc/pgm/points/SpreadPointProvider.java

```diff
@@ -37,14 +40,14 @@ public Location getPoint(Match match, @Nullable Entity entity) {
           }
         }
 
-        if (bestDistance < nearest) {
+        if (bestDistance <= nearest) {
           bestDistance = nearest;
-          bestPoint = pos;
+          bestPoints.add(pos);
         }
```

Comment:
`if (bestDistance < nearest)`, you need to clear the list of bestPoints, otherwise you will be adding to the list all points wich become better, but without removing previous worse points.